cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/MetricSamplerPartitionAssignor.java

```diff
@@ -17,13 +17,14 @@
 public interface MetricSamplerPartitionAssignor extends CruiseControlConfigurable {
 
   /**
-   * @deprecated Please use {@link #assignPartitions(Cluster)}.
+   * Please use {@link #assignPartitions(Cluster)}.
```

Comment:
Nit: Should we keep `@deprecated` tag?

Reply:
this was to solve a compilation warning - iiuc the `@Deprecated` annotation needs to precede the method declaration and not embedded in the comment - so moved it here: https://github.com/linkedin/cruise-control/pull/1217/files#diff-71850ea35e26c703f1a1253519e0c41cR27

Reply:
Cool - makes sense. Let me add it back to javadoc comment